projects/BFDR/IJEFetalDeath.cs

```diff
@@ -11,12 +11,12 @@ namespace BFDR
     /// FHIR based <c>FetalDeathRecord</c> to the IJE format for a specific field, and
     /// the setters convert from IJE format for a specific field and set that value
     /// on the embedded FHIR based <c>FetalDeathRecord</c>.</summary>
-    public class IJEFetalDeath : IJE
+    public class IJEFetalDeath : IJENatality
     {
         private readonly FetalDeathRecord record;
 
         /// <summary>Constructor that takes a <c>FetalDeathRecord</c>.</summary>
-        public IJEFetalDeath(FetalDeathRecord record, bool validate = true)
+        public IJEFetalDeath(FetalDeathRecord record, bool validate = true) : base(record, validate)
```

Comment:
I think the contents of this Constructor could be moved to the base constructor in IJE Natality.

Reply:
Ditto for the 2 constructors that just have "ProcessIJE()"